### PR TITLE
SAK-44878 - Mark the LTI Advantage JSON documents as UTF-8

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

**Comment by jonespm** [comment]
@ottenhoff previously approved this on #8973, re-opened because of jira issue.

File: basiclti/tsugi-util/src/java/org/tsugi/ags2/objects/Result.java [hunk 1]
```diff
@@ -28,6 +28,9 @@
 	public static final String MIME_TYPE = "application/vnd.ims.lis.v2.result+json";
 	public static final String MIME_TYPE_CONTAINER = "application/vnd.ims.lis.v2.resultcontainer+json";
 
+	public static final String CONTENT_TYPE = MIME_TYPE + "; charset=utf-8";
```
Comment (jonespm) [inline basiclti/tsugi-util/src/java/org/tsugi/ags2/objects/Result.java:31]: Is there a reason for having these same constants in `LineItem` and `Result`? **Looks** like `LTI13Servlet` is the only place these are used and they're just the same. Are these ever different? Seems like they could just be in `LTI13ConstantsUtil`?

**Comment by jonespm** [comment]
I think the build is failing because the branch is out of date, I can't force push to this branch.

File: basiclti/tsugi-util/src/java/org/tsugi/ags2/objects/Result.java (hunk 1 above)
Comment (csev) [reply basiclti/tsugi-util/src/java/org/tsugi/ags2/objects/Result.java:31]: Matt are we both working on this?

Comment (csev) [reply basiclti/tsugi-util/src/java/org/tsugi/ags2/objects/Result.java:31]: They are not the same. One is "lineitem" and the other is "result".  Where things belong in LTI depends on what is going on.   Everything in the basiclti folder that is "tsugi-util" is a literal translation of the IMS specs into generic (not Sakai specific) java code.  

LTI13Servlet is in basiclti-blis - which is Sakai specific.  It uses these spec defined constants.

The tsugi-util code is separately released in Sonatype as

<dependency>
   <groupId>org.tsugi</groupId>
   <artifactId>tsugi-util</artifactId>
   <version>0.4-SNAPSHOT</version>
</dependency>

**Comment by csev** [comment]
Matt - I rebased and force pushed so it is restarting.

**Comment by csev** [comment]
@jonespm - I think that @ern broke master with one of his TaskService commits - it is where the build is failing with my patch - and I have rebased it and force pushed it and the compile ran again.

**Comment by csev** [comment]
Master dies on my laptop when I recompile with unit tests on.

```
[ERROR] Errors: 
[ERROR]   TaskServiceTest.testCanCreateSingleUserTask » IllegalState Failed to load Appl...
[ERROR]   TaskServiceTest.testCanCreateTask » IllegalState Failed to load ApplicationCon...
[ERROR]   TaskServiceTest.testCanDeleteTask » IllegalState Failed to load ApplicationCon...
[ERROR]   TaskServiceTest.testCanDeleteTaskByReference » IllegalState Failed to load App...
[ERROR]   TaskServiceTest.testCanRemoveUserTask » IllegalState Failed to load Applicatio...
[ERROR]   TaskServiceTest.testCanSaveUserTask » IllegalState Failed to load ApplicationC...
[ERROR]   TaskServiceTest.testCanSoftDeleteTask » IllegalState Failed to load Applicatio...
```

**Comment by jonespm** [comment]
Thanks, I tried to push to your branch didn't have permission.

Looks like this No-PR commit @ern did on 0170595cf4f48413922dffe26bdb04410c1bb359

Thought it would have been something fixed in master, but nope it was still broke in master. I fixed this tonight in #8978 so should be merged tomorrow.

**Comment by csev** [comment]
Matt - I re based this and dit a git push to the branch so it will rebuild with the fix to master.

**Comment by jonespm** [comment]
I was talking to Earle about making this process a little easier/seamless for the committers. I think he started on something but didn't seem to be working as expected.

Like maybe adding in:
Auto-rebase label: https://github.com/marketplace/actions/auto-rebase
Or a action to rebase: https://github.com/cirrus-actions/rebase

It should have been taken care of already but there was some issue with the checkout. 

Thanks! Looks like it passed now.

**Review by csev — APPROVED** [review]

**Pull request merged**